### ISSUE TITLE
Puzzle block position out of picture size

### ISSUE THREAD

**Issue by hutomosaktikartiko** [BrianTV98/slider_captcha#11]
![Simulator Screen Shot - iPhone 12 - 2022-08-04 at 15 12 40](https://user-images.githubusercontent.com/51199036/182798111-289b6c63-656f-4e4e-ae37-a3cc5033c4b5.png)

This is my code:
```
Scaffold(
      body: SafeArea(
        child: Column(
          // mainAxisSize: MainAxisSize.min,
          mainAxisAlignment: MainAxisAlignment.center,
          children: [
            SliderCaptcha(
              controller: controller,
              image: Image.asset(
                'assets/images/Selfie-1.png',
                fit: BoxFit.fitWidth,
              ),
              colorBar: Colors.blue,
              colorCaptChar: Colors.green,
              onConfirm: (value) {
                Future.delayed(const Duration(seconds: 1)).then(
                  (value) {
                    controller.create();
                  },
                );
              },
            ),
          ],
        ),
      ),
    );
```

**Comment by BrianTV98** [comment]
Hi @hutomosaktikartiko  
I fixed the problem you mentioned. Thank you for contributing issues to make the library more complete.

**Comment by hutomosaktikartiko** [comment]
Thanks for the fix, but I find some problem again. If I give padding or wrap with AlerDialog widget the SliderCaptcha look good, but result value (bool) from onConfirm always return false. This is my code:
```
    showDialog(
      context: context,
      builder: (context) {
        return AlertDialog(
          shape:
              RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
          contentPadding: EdgeInsets.zero,
          content: Column(
            mainAxisSize: MainAxisSize.min,
            mainAxisAlignment: MainAxisAlignment.center,
            children: [
              ClipRRect(
                borderRadius: BorderRadius.circular(10),
                child: SliderCaptcha(
                  controller: controller,
                  image: Image.asset(
                    'assets/images/Liveness.png',
                    fit: BoxFit.contain,
                    width: SizeConfig.screenWidth,
                  ),
                  colorBar: Colors.blue,
                  colorCaptChar: Colors.blue,
                  onConfirm: (value) {
                    if (!value) {
                      controller.create();
                    }
                    print(value);
                  },
                ),
              ),
            ],
          ),
        );
      },
    );
```

**Comment by BrianTV98** [comment]
i found the cause of the issue you mentioned. This is really a serious
problem. puzzle block is using its coordinates relative to the screen
instead of its coordinates in the parent widget. so now it only works
properly when you use full screen.
I will fix this problem. however it can take quite a while.

**Comment by BrianTV98** [comment]
> 

I fixed this issue at version 0.1.4. Please check again and feedback if has error

**Comment by BrianTV98** [comment]
> Thanks for the fix, but I find some problem again. If I give padding or wrap with AlerDialog widget the SliderCaptcha look good, but result value (bool) from onConfirm always return false. This is my code:
> 
> ```
>     showDialog(
>       context: context,
>       builder: (context) {
>         return AlertDialog(
>           shape:
>               RoundedRectangleBorder(borderRadius: BorderRadius.circular(10)),
>           contentPadding: EdgeInsets.zero,
>           content: Column(
>             mainAxisSize: MainAxisSize.min,
>             mainAxisAlignment: MainAxisAlignment.center,
>             children: [
>               ClipRRect(
>                 borderRadius: BorderRadius.circular(10),
>                 child: SliderCaptcha(
>                   controller: controller,
>                   image: Image.asset(
>                     'assets/images/Liveness.png',
>                     fit: BoxFit.contain,
>                     width: SizeConfig.screenWidth,
>                   ),
>                   colorBar: Colors.blue,
>                   colorCaptChar: Colors.blue,
>                   onConfirm: (value) {
>                     if (!value) {
>                       controller.create();
>                     }
>                     print(value);
>                   },
>                 ),
>               ),
>             ],
>           ),
>         );
>       },
>     );
> ```
I fixed this issue at version 0.1.4. Please check again and feedback if has error
